### PR TITLE
Fixed Issue #272, sulu.rlp.part can be used multiple times (but without priority for now)

### LINKED ISSUE #272: sulu.rlp.part can be used multiple times
Can you add a description in the docs ...
http://docs.sulu.io/en/latest/reference/content-types/resource_locator.html
... that sulu.rlp.part can be used multiple times and the field values will be concatenated!?

Thanks!

### PR DESCRIPTION
**Pull request by baroso**
| Q | A
| --- | ---
| Fixed tickets | fixes #272 
| License | MIT

#### What's in this PR?

Explanation and code example.

#### Why?

Added documentation about sulu.rlp.part, that it can be used multiple times and the values are concatenated.

### CODE REVIEW

File: reference/content-types/resource_locator.rst [hunk 1]
```diff
@@ -19,10 +19,11 @@ Tags
...
       - Description
     * - sulu.rlp
-      - The resource locator with this tag defines the URL to a specific page
+      - The resource locator with this tag defines the URL to a specific page.
     * - sulu.rlp.part
-      - Fields marked with this tag are used to generate the URL for a specific
-        page
+      - Fields marked with this tag are used to generate the URL for a specific page.
+        If more than one field ist marked, the values of these fields will be concatenated into the resource locator.
+        The sequence depends on the value of the priority parameter.
```
Comment (danrot) [inline reference/content-types/resource_locator.rst]: Sadly that's not true, this is a bug at the moment 😕 I think as long as it is not working we should not have it like this in the documentation.

Comment (danrot) [inline reference/content-types/resource_locator.rst]: Sadly that's not true, this is a bug at the moment 😕 I think as long as it is not working we should not have it like this in the documentation.

Comment (baroso) [inline reference/content-types/resource_locator.rst]: No problem, that's what I meant with my "should I wait" comment: https://github.com/sulu/sulu-docs/issues/272#issuecomment-248013989 ;)

Comment (danrot) [inline reference/content-types/resource_locator.rst]: Ah, ok, I was talking about something different 😅 We should change the documentation now, but omit the `priority` for the time being.

Comment (baroso) [inline reference/content-types/resource_locator.rst]: Did I do this right with my second commit?

Comment (danrot) [inline reference/content-types/resource_locator.rst]: Do it in the same PR. Ideally you would rebase the existing commit, but a separate commit is also fine.

**Pull request merged**